tests/Doctrine/Tests/TestUtil.php

```diff
@@ -213,11 +193,6 @@ private static function mapConnectionParameters(array $configuration, string $pr
         return $parameters;
     }
 
-    public static function getPrivilegedConnection(): Connection
```

Comment:
Why is this change needed?

Reply:
Again just to bring this file in sync with the doctrine/orm equivalent - the method has not been removed, just moved up the file so that the diff between the two projects only contains expected / intentional differences.

Reply:
It makes the review harder. There are components in existing coding standards that enforce the or of members in the class. Before it's standardized, there's no guarantee that it will remain like this, so let's not make this change now.

Reply:
OK but presumably when those are standardized they'll be applied across both ORM and DBAL so will produce the same change in both. And will result in a massive diff like the recent coding standards ones anyway, so this method moving will be fairly irrelevant in the bigger picture.

For me as an outsider, trying to debug why the ORM CI was not working as expected when DBAL was, the huge diff between the TestUtil classes in each project meant it took me a lot longer to spot the problem. And I suspect contributed to the issue arising in the first place since it wouldn't have been easy to see that the two projects couldn't just use the same phpunit / CI config files.

Hence I think it makes a lot more sense to minimise the diff between projects even if it means a few extra LOC changed in this PR.

But if you'd like me to move it back I can.